docs/releases/3.1.txt

```diff
@@ -225,6 +225,12 @@ Validators
 
 * ...
 
+Miscellaneous
+~~~~~~~~~~~~~
+
+* The SQLite backend now supports :class:`pathlib.Path` for the ``NAME``
```

Comment:
The change to `BASE_DIR` should also be highlighted somewhere to encourage people to use `pathlib` in existing projects. This is because the documentation will now show examples using it instead of `os.path` functions. I think this is better in the long run, however, because it is easier for novices than lots of nested functions.